hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1512,6 +1512,7 @@ func (r *HostedClusterReconciler) reconcileControlPlaneOperator(ctx context.Cont
 		return fmt.Errorf("failed to reconcile controlplane operator rolebinding: %w", err)
 	}
 
+	// TODO: Remove this block after initial merge of this feature. It is not needed for latest CPO version
```

Comment:
are we saying that the latest hypershift can only deploy guest clusters 4.11+ ? otherwise it seems like we'd need this for CPO's tied to release payloads < 4.11